ow_lander/src/ow_lander/actions.py

```diff
@@ -225,7 +225,7 @@ def ground_detect_cb():
         self._pub_result.publish(False, '', Point())
         self._set_succeeded("No ground detected", final=Point(), success=False)
     finally:
-      self._arm.move_group_scoop.set_planner_id('RRTstar')
+      self._arm.move_group_scoop.set_planner_id('RRTConnect')
```

Comment:
I don't know the implications of changing the planner.  Is this safe to do without more extensive testing?

Reply:
I'm pretty confident it's safe. It should've always switched back to RRTConnect, the same as the other RRTStar using actions do. Things don't break in an obvious way without this change, you just suddenly have all of your arm trajectories being planned with RRTstar instead of RRTConnect. It's a silent, but fundamental change to arm operation that would have been very easy to trip during a simulation.

Reply:
Anjan had experimented a lot with both these planners and had strong reasons for one over the other.  I forget the details, but for a while we switched to one and saw much faster planning but other problems including poor trajectories, and then he switched some actions back to the original planner.  The history and rationale should be found in closed PRs and Jiras.